jobs/exemple_update_game_datetime.py

```diff
@@ -0,0 +1,37 @@
+import sqlite3
+"""Script à modifier suivant la situation, et à executer dans le venv du serveur
+> cd lolprono
+> source venv/bin/activate
+> python
+"""
+def create_connection(db_file):
+    try:
+        conn = sqlite3.connect(db_file)
+        return conn
+    except sqlite3.Error as e:
+        print(e)
+        return None
+
+
+def update_developer_salary(conn):
```

Comment:
On est vraiment payés ????? 
```suggestion
def update_game_datetime(conn):
```